src/iosrv/init.c

```diff
@@ -614,6 +614,12 @@ parse(int argc, char **argv)
 			dss_core_offset = nr;
 			break;
 		case 'g':
+			if (strlen(optarg) > sizeof(uuid_t) - 1) {
```

Comment:
why sizeof(uuid_t) ?   The group name is just a string for the group id.  Seems this should be based on GROUP_ID_MAX_LEN

Reply:
The group name is becoming the system name, which will be cast to uuid_t.